CHANGELOG.next.md

```diff
@@ -74,6 +74,8 @@ Thanks, you're awesome :-) -->
 
 ## Unreleased
 
+* Added `file.fork_name` field #1288
```

Comment:
Can you bump this up under `#### Added` at line 23?

Reply:
199c64a